.storybook/main.js

```diff
@@ -0,0 +1,48 @@
+const path = require("path");
+
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials"
+  ],
+  "core": {
+    "builder": "webpack5"
+  },
+  "webpackFinal": async (config, { configType }) => {
+    // `configType` has a value of 'DEVELOPMENT' or 'PRODUCTION'
+    // You can change the configuration based on that.
+    // 'PRODUCTION' is used when building the static version of storybook.
+
+    // Make whatever fine-grained changes you need
+    // TODO: Create a custom webpack.config.js if this gets too long
+    // ? https://storybook.js.org/docs/react/configure/webpack#using-your-existing-config
+    config.module.rules.push(
+      {
+        test: /\.scss$/,
+        use: ['style-loader', 'css-loader', 'sass-loader'],
+        include: path.resolve(__dirname, '../'),
+      },
+      {
+        test: /.(jpe?g|png|svg)$/,
+        loader: "url-loader",
+        include: path.join(__dirname, "src/img")
+      },
+      {
+        test: /\.twig$/,
```

Comment:
I wonder if we are going to be able to set up lit-html (web components) with this storybook base, or if it is better to go the other way?

Reply:
was able to use a conditional `resourceQuery` and call separate loaders for the scss files. Maybe there is another way to do this, but I think it at least gives us a POC on how we can build web-components and the twig templates in the same storybook project 👍 

https://github.com/uswds/uswds/blob/jm-add-storybook-lithtml-test/.storybook/main.js#L26-L43

```js
        "oneOf": [
          {
            "resourceQuery": /wc/, // foo.scss?wc
            "use": [{
              loader: 'lit-scss-loader',
                options: {
                  minify: true,
                },
              },
              "sass-loader",
            ],
          },
          {
            "use": ['style-loader', 'css-loader', 'sass-loader'],
            "include": path.resolve(__dirname, '../'),
          }
        ],
      },
```

Reply:
This is great! I think this will be the best route, for now.